### PR TITLE
Follow-up edits to PR#3229

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/3229

Preview Builds:
http://file.rdu.redhat.com/~ahardin/11162016/PR3229-followup/install_config/router/default_haproxy_router.html#using-wildcard-routes

http://file.rdu.redhat.com/~ahardin/11162016/PR3229-followup/architecture/core_concepts/routes.html#wildcard-subdomain-route-policy

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -805,17 +805,17 @@ WebSocket connections to timeout frequently on that route.
 
 
 [[wildcard-subdomain-route-policy]]
-== Creating routes specifying a wildcard subdomain policy
+== Creating Routes Specifying a Wildcard Subdomain Policy
```
Comment (adellape) [inline architecture/core_concepts/routes.adoc]: Is "Creating Routes to Specify [...]" any better? :thinking: May be fine as is.

**Comment by ahardin-rh** [comment]
@ramr @knobunc PTAL

**Review by knobunc — APPROVED** [review]
LGTM.  I like the new procedure of doing the merge and then the follow-on PR.

**Review by adellape — COMMENTED** [review]
LGTM

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by ahardin-rh** [comment]
Revision history in original PR

**Pull request merged**